2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -244,7 +244,16 @@ function setup_keyfile() {
     exit 0
```

Comment:
@omron93 The `exit 0` here looks very suspicious for me. As far I understand it should be replaced by `return`, right? Because if we execute `exit` inside of the function it will exit the script and other code won't be executed.